lib/openshift/project.rb

```diff
@@ -21,7 +21,7 @@ def visible?(user: nil, result: {}, quiet: false)
         return true
       else
         case  result[:response]
-        when /cannot get project/, /not found/
+        when /cannot get project/, /not found/, /Error from server \(Forbidden\)/
```

Comment:
we can remove old messages because this branch is for v4.0+, no need to keep old baggage

Reply:
Yes, removed.